public/styles/share_mtr.css

```diff
@@ -1,6 +1,6 @@
+    src: url('../cdn/Vegur-Regular.otf');
```

Comment:
Using relative link to font may cause the exported SVG fails to load font. It's better to keep the base64 data uri.

Reply:
It explains the reason why we use base64, changing back to the previous code.